rundeck-data-model/src/main/java/org/rundeck/app/data/model/v1/execution/RdReferencedExecution.java

```diff
@@ -0,0 +1,12 @@
+package org.rundeck.app.data.model.v1.execution;
+
+
+import java.io.Serializable;
+
+public interface RdReferencedExecution {
+
+    String getStatus();
+    Serializable getExecutionId();
+    Serializable getJobId();
```

Comment:
We want to associate ReferencedExecution's to job's using the job uuid.

See https://github.com/rundeck/rundeck/pull/8226/files#diff-0b7f53ad4158de3da201d34917fa00415291644bd320eaf011ecdaeef7f43548 for how I did it in my PR.